content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -207,33 +270,50 @@ The Go tracer includes support for the following data stores and libraries.
+| [GraphQL (graph-gophers)][26]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/graph-gophers/graphql-go][27]          |
+| [GraphQL (graphql-go)][87]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/graphql-go/graphql][88]          |
+| [GraphQL (gqlgen)][89] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen][90]                 |
 | [HTTP][28]              | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http][29]                          |
 | [HTTP router][30]       | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/julienschmidt/httprouter][31]          |
 | [Redis (go-redis)][32]  | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis][33]                    |
-| [Redis (go-redis-v8)][34]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v8][35]                |
+| [Redis (go-redis-v7)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v7][91]                |
+| [Redis (go-redis-v8)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v8][35]                |
+| [Redis (go-redis-v9)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/redis/go-redis.v9][92]                |
 | [Redis (redigo)][36]    | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/garyburd/redigo][37]                   |
 | [Redis (new redigo)][38]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gomodule/redigo][39]                   |
+| [Redis (rueidis)][93]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/redis/rueidis][94]                   |
 | [SQL][40]               | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql][41]                      |
 | [SQLx][42]              | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/jmoiron/sqlx][43]                      |
+| [PostgreSQL (pgx v5)][95] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/jackc/pgx.v5/v2][96]                  |
 | [MongoDB][44]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go.mongodb.org/mongo-driver/mongo][45] |
 | [MongoDB (mgo)][114]    | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/globalsign/mgo][46]                    |
 | [BuntDB][47]            | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/tidwall/buntdb][48]                    |
 | [LevelDB][49]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/syndtr/goleveldb/leveldb][50]          |
 | [miekg/dns][51]         | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/miekg/dns][52]                         |
-| [Kafka (confluent)][53] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/confluentinc/confluent-kafka-go][54]   |
+| [Kafka (confluent)][53] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/confluentinc/confluent-kafka-go/kafka][54]   |
```

Comment:
This link text doesn't match the link that's opened (there's no `/kafka` at the end). Could you please confirm if this is expected, and update if needed?